tests/helper/skyMapTestCase.py

```diff
@@ -30,33 +32,63 @@
 class SkyMapTestCase(lsst.utils.tests.TestCase):
     """An abstract base class for testing a SkyMap.
 
-    To use, subclass and set the following class variables:
-    _SkyMapClass: the particular SkyMap subclass to test
-    _SkyMapConfig: a SkyMapConfig instance to use in the test, or None to use _SkyMapClass.ConfigClass()
-    _SkyMapName: the name of the particular SkyMap class in the registry
-    _NumTracts: the number of tracts to expect (for the default configuration)
-    _NeighborAngularSeparation: Expected angular separation between tracts
-    _numNeighbors: Number of neighbors that should be within the expected angular separation
+    To use, subclass and call `setAttributes` from `setUp`
     """
-    _SkyMapConfig = None
+    def setAttributes(self, *,
```

Comment:
I don't recognise this syntax. Is it py3? What's it mean?

Reply:
Yes it is py3; all arguments following a bare asterisk can only be specified by name.